CHANGELOG.md

```diff
@@ -6,19 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [6.1.0] 2020-06-30
+## [v6.2.0] 2020-07-29
+## Added
+- CI pipeline
+  - build
+  - on release, publish docker image and github release with binaries
+
+## [v6.1.0] 2020-06-30
 ## Changed
 - Dockerfile labels
 - Dockerfile src image
 
-# Added
+## Added
 - SLO computation recording rules, alerts
 - slo-exporter grafana dashboard
 
 ## Fixed
 - PrometheusIngester: Fixed unit of the `slo_exporter_prometheus_ingester_query_duration_seconds_bucket` metric.
 
-## [6.0.0] 2020-06-12
+## [v6.0.0] 2020-06-12
```

Comment:
Just form curiosity why are we changing these?

Reply:
I would like to unify format of `git tags`, here mentioned`changelog headers` and docker tags.